Modules/Filtering/QuadEdgeMeshFiltering/include/itkQuadEdgeMeshDecimationCriteria.h

```diff
@@ -135,7 +135,9 @@ class ITK_TEMPLATE_EXPORT NumberOfPointsCriterion
   using typename Superclass::PriorityType;
 
   inline bool
-  is_satisfied(MeshType * iMesh, const ElementType & itkNotUsed(iElement), const MeasureType & itkNotUsed(iValue)) const override
+  is_satisfied(MeshType *          iMesh,
+               const ElementType & itkNotUsed(iElement),
+               const MeasureType & itkNotUsed(iValue)) const override
```

Comment:
Fine by me, just wondering: are those formatting changes caused by the clang-format update (from v19.1.4 to v19.1.7) that you are proposing with this pull request?

Reply:
No, I do not think so.

There were style changes inconsistent with clang-format because our ClangFormat GitHub Action was removed.

I did the update to get consistent behavior locally and in CI.